lib/macho/macho_file.rb

```diff
@@ -424,14 +478,24 @@ def low_fileoff
       offset
     end
 
+    # Updates the number of load commands in the raw data.
+    # @param ncmds [Fixnum] the new number of commands
+    # @return [void]
+    # @private
+    def set_ncmds(ncmds)
```

Comment:
Thanks for the feedback! I've pondered this a bit more, and I'm currently leaning more towards leaving it as-is or renaming to `update_ncmds`. The problem with making it `ncmds=` is that I would expect `ncmds` to immediately return the new value, but this is not the case because the setter doesn't currently update it. The update is delayed until all modifications have been made and the calling method hopefully remembered to trigger an update of everything from `@raw_data`.